### PR TITLE
Update fed ramp c2 cs docs

### PR DESCRIPTION
**Pull request by yleiferman**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

https://jasklabs.atlassian.net/browse/CON-1275

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ sidebar_label: Cisco AMP
 The Cisco AMP Source provides a secure endpoint to receive data from the Cisco Amp [System Log API](https://api-docs.amp.cisco.com/api_resources?api_host=api.amp.cisco.com&api_version=v1). It securely stores the required authentication, scheduling, and state tracking information.
 
 :::note
-This Source is not available in the Fed deployment.
+This Source is available in the Fed deployment.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md:11]: @yleiferman , should this say FedRAMP deployment?

**Review by kimsauce — CHANGES_REQUESTED** [review]
One question (applies to all docs)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md (hunk 1 above)
Comment (yleiferman) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md:11]: Hi @kimsauce , that's a good question. The official name of the deployment is fed. FedRAMP is a certification from the federal government.

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md:11]: Oh I see - thanks for clarifying! Are you ok if I hyperlink these notes with the Fed deployment docs section?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ keywords:
 The Okta Source provides a secure endpoint to receive event data from the Okta [System Log API](https://developer.okta.com/docs/reference/api/system-log/) and [Users API](https://developer.okta.com/docs/reference/api/users/).
 It securely stores the required authentication, scheduling, and state tracking information.
 
-This Source is not available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints). Use the [legacy collection method for Okta](/docs/integrations/saml/Okta#Collecting-logs-for-Okta).
+:::note
+This Source is available in the Fed deployment.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source.md:14]:
```suggestion
This Source is available in the [Fed deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/salesforce-source.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ sidebar_label: Salesforce
 
 The Salesforce Source provides a secure endpoint to receive event data from the Salesforce through its [Rest API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm). The source securely stores the required authentication, scheduling, and state tracking information.
 
+:::note
+This Source is available in the Fed deployment.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/salesforce-source.md:10]:
```suggestion
This Source is available in the [Fed deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ The Tenable Source provides a secure endpoint to ingest audit-log events, vulner
    * The Asset Export API first [exports assets](https://developer.tenable.com/reference/exports-assets-request-export) that are used to initiate export jobs. Next, it gets the export [status](https://developer.tenable.com/reference/exports-assets-request-export) and then [downloads exported assets](https://developer.tenable.com/reference/exports-assets-download-chunk) in a chunk.
 
 :::note
-This Source is not available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints).
+This Source is available in the Fed deployment.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/tenable-source.md:17]:
```suggestion
This Source is available in the [Fed deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hyperlinked "Fed deployment"

**Review by kimsauce — APPROVED** [review]
Hyperlinked "Fed deployment"

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md (hunk 1 above)
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md:11]:
```suggestion
This Source is available in the [Fed deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security)
```

**Pull request merged**